### PR TITLE
force redirect to https

### PR DESCRIPTION
**Pull request by peerwit**

### CODE REVIEW

**Comment by scottrice10** [comment]
:+1:

File: server/api/amazon/index.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function test(req, res, next){
 	opHelper.execute('ItemSearch', {
 	  'Keywords': req.body.term,
 	  'SearchIndex': 'Blended',
-	  'ResponseGroup': 'Similarities,ItemIds'
+	  'ResponseGroup': 'Similarities,ItemIds,Images,Small'
 	}, function(err, results) {
 		console.log(results);
```
Comment (kxprim) [inline server/api/amazon/index.js]: Indentation to correct

**Comment by kxprim** [comment]
Can  :+1:  once indentation is corrected

File: server/api/amazon/index.js [hunk 2]
```diff
@@ -17,14 +17,14 @@ function test(req, res, next){
 	});
 	console.log(config.amazon.clientID);
 	opHelper.execute('ItemSearch', {
-	  'Keywords': req.body.term,
-	  'SearchIndex': 'Blended',
-	  'ResponseGroup': 'Similarities,ItemIds'
-	}, function(err, results) {
-		console.log(results);
+		  'Keywords': req.body.term,
```
Comment (kxprim) [inline server/api/amazon/index.js]: Indentation to fix

**Comment by scottrice10** [comment]
:+1:

**Comment by kxprim** [comment]
:+1:  All good

**Pull request merged**